CHANGELOG.md

```diff
@@ -50,5 +52,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - This CHANGELOG file
 - Tagging first version
 
-[unreleased]: https://github.com/giantswarm/g8s-prometheus/compare/v0.1.0...HEAD
```

Comment:
Wait wtf happened here

Reply:
Well, this has always been wrong all this time